backend/app/assets/javascripts/spree/backend/models/image_upload.js

```diff
@@ -56,7 +56,7 @@ Spree.Models.ImageUpload = Backbone.Model.extend({
       processData: false,  // tell jQuery not to process the data
       contentType: false,  // tell jQuery not to set contentType
       xhr: function () {
-        var xhr = $.ajaxSettings.xhr();
+        var xhr = $.ajaxSetup.xhr(); // Using default ajax builder but inputting upload settings
```

Comment:
🙏 thanks for this!

Reply:
Unsure how this is working for anyone. We're getting errors about `ajaxSetup.xhr` being undefined, and I cannot find references to it in anywhere else in Solidus, or in any of our external libraries including jQuery itself.

Where is `ajaxSetup.xhr` being defined?